aws-ecs/README.md

```diff
@@ -0,0 +1,61 @@
+
+## Exporting AWS ECS Stats to Wavefront using Telegraf
+
+Amazon ECS, input plugin uses the Amazon ECS metadata and stats v2 or [v3][task-metadata-endpoint-v3] API endpoints to gather stats on running containers in a Task.
+
+This task makes use of telegraf-ecs plugin and export stats to Wavefront (https://wavefront.com).
+
+The telegraf agent running on ECS instance enables the input plugin and the output plugin which are ECS and Wavefront respectively.
+
+```
+ [[inputs.ecs]]
+ 
+ [[outputs.wavefront]]
+```
+For further reference, please refer to the documentation - https://github.com/influxdata/telegraf/blob/release-1.18/plugins/inputs/ecs/README.md
```

Comment:
Change line number 15 to:- For further reference, please refer to the telegraf [ecs plugin](https://github.com/influxdata/telegraf/blob/release-1.18/plugins/inputs/ecs/README.md)